src/Model/Block.php

```diff
@@ -106,6 +111,8 @@ public function rotate(): self
         $this->setWidth($this->getHeight());
         $this->setHeight($w);
 
+        $this->rotated = true;
```

Comment:
`$this->rotated = !$this->rotated;`

I'd just negate it. That way, if rotate is called multiple times for some reason, we still know which is the original.